src/components/DashboardPage/ProfileDashboardData.js

```diff
@@ -2,17 +2,20 @@ import React, { Component } from 'react';
 import { PROFILE_ACTIONS } from '../../redux/actions/profileActions';
 
 const mapStateToProps = reduxState => ({
-    profile: reduxState.profile
+    profileReducer: reduxState.profileReducer
 });
 class ProfileDashboardData extends Component {
+    componentDidMount() {
+        this.props.dispatch({ type: PROFILE_ACTIONS.FETCH_PROFILES })
+    }
     render() {
     
     return (
       <div>
-          Potential Spam Profiles : {this.props.profile.length}
+          Potential Spam Profiles : {this.props.profileReducer.length}
```

Comment:
profileReducer is an object, not an array, so it's doesn't have a .length method that gives meaningful information